android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/NearbyBeaconsFragment.java

```diff
@@ -192,7 +221,7 @@ public void onPrepareOptionsMenu(Menu menu) {
   @Override
   public void onListItemClick(ListView l, View v, int position, long id) {
     // If we are scanning
-    if (mIsScanRunning) {
+    if (mScanningAnimationDrawable.isRunning()) {
```

Comment:
Right, the animation should only be running when there are no items in the list.  The full scan goes for 10 seconds now, not just 3, so we need to let people interact with the listview before then.